src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -302,6 +302,43 @@ static void Main()
             Enumerable.Range(0, 1).Sum(f =>
             Enumerable.Range(0, 1).Count(g => true)))))));
     }
+}";
+            var comp = CreateCompilation(source);
+            comp.VerifyDiagnostics();
+        }
+
+        [ConditionalFactAttribute(typeof(IsRelease))]
+        [WorkItem(1083969, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1083969")]
+        public void NestedLambdas_02()
```

Comment:
>NestedLambdas_02 [](start = 20, length = 16)

What exactly are we testing here? Were we unable to compile the code before?

Reply:
We're testing performance. On my machine, this test completes in a couple of seconds with this change and 15+ mins without.

---
In reply to: [410978595](https://github.com/dotnet/roslyn/pull/43366#discussion_r410978595) [](ancestors = 410978595)

Reply:
> We're testing performance. On my machine, this test completes in a couple of seconds with this change and 15+ mins without.

Consider adding a comment to make it easy to analyze failures.

---
In reply to: [411004957](https://github.com/dotnet/roslyn/pull/43366#discussion_r411004957) [](ancestors = 411004957,410978595)